.gitignore

```diff
@@ -16,3 +16,4 @@ build/
 
 # Marquez configuration
 config.yml
+/.metadata/
```

Comment:
You'll want to define a [`.gitignore_global`](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) to ignore files specific to your dev environment. The `.gitignore` in this repo serves as excluding files specific to the project.